dateparser/custom_lang_detect/fast_text.py

```diff
@@ -0,0 +1,37 @@
+try:
+    import fasttext
+except:
+    import sys
+    import subprocess
+    print("Installing fast_text library")
+    subprocess.check_call([sys.executable, '-m', 'pip', 'install','fasttext'])
+    import fasttext
+
+import os 
+dir_path = os.path.dirname(os.path.realpath(__file__))
+
+model = "lid.176.ftz"
+_model_path = dir_path + '/data/' + model
+
+def check_model():
+    if os.path.exists(_model_path) == False:
+        import urllib.request
+        print("Downloading model", model)
+        url = "https://dl.fbaipublicfiles.com/fasttext/supervised-models/" + model
+        urllib.request.urlretrieve(url, _model_path)
+
+check_model()
+
+_language_parser = fasttext.load_model(_model_path)
+_CONFIDENCE_THRESHOLD = 0.5
+
+def language_parser(text):
```

Comment:
as this is a function (not a class), the name should be a verb, so I would change it to something like 

`parse_languages` or `detect_languages` 

(and I also would put a plural because it returns multiple matches).